esmvaltool/diag_scripts/weighting/climwip.py

```diff
@@ -422,15 +425,38 @@ def main(cfg):
     """Perform climwip weighting method."""
     models, observations = read_metadata(cfg)
 
-    variable_groups = models.keys()
+    variable_groups_independence = [
+        key for key, value in cfg['independence_contributions'].items()
+        if value > 0
+    ]
+    variable_groups_performance = [
+        key for key, value in cfg['performance_contributions'].items()
+        if value > 0
```

Comment:
Is this check for > 0 really necessary? I guess the main difference is that they will not show up in the intermediate output plots now, is that it? But then, shouldn't we also have a way to visualize the contributions of each variable group in those figures?

Reply:
In my opinion it is cleaner to treat contribution=0 the same way as not including a variable at all. Otherwise this becomes a strange hybrid case of variable groups showing up in the individual plots but not in the mean plot. If we really want all the intermediate plots we can run a dedicated recipe with everything we want, right?